src/screens/DiscoveryScreenV2/fragment/UsersFragment.js

```diff
@@ -345,7 +357,7 @@ const UsersFragment = ({
     );
   return (
     <View>
-      {loadingDM && (
+      {(loadingDM || loadingCreateChat) && (
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The condition for rendering the loading view has been updated to include `loadingCreateChat`. This is a good change as it ensures that the loading view is displayed whenever a chat is being created, improving the user experience.
```diff
-      {loadingDM && (
+      {(loadingDM || loadingCreateChat) && (
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->